### ISSUE TITLE
resourceGroupField event constraint

### ISSUE THREAD

**Issue by emcvey-administrate** [fullcalendar/fullcalendar#6892]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

We're using a timeline view (gantt chart). The resources on our timeline view are divided into a few different types--arbitrary example but think classroom, TA, and professor. Each event will have one resource of each type using the `resourceIds` property. We want to allow them to use the drag n drop feature to change the assigned resources, but we want to limit it the same type of resource. So they can change from TA Tommy to TA Trisha, but they can't change a classroom to a TA.

I can't currently achieve this with `eventAllow` because while it does give the resource info for the drop location, it doesn't expose any info about the previous location--it gives you the event as the whole, but that contains all three resources. 

I am requesting that either the previous event resource is exposed, or a constraint for `resourceGroupField` is added.

**Comment by acerix** [comment]
You can get the previous resource of the event from `eventAllow` with `draggedEvent.getResources()`.  If I understand, that should work to keep in the same resource type like this:

https://codepen.io/acerix/pen/vYdMLmq?editors=001

**Comment by emcvey-administrate** [comment]
@acerix that doesn't work for my scenario because all events have 3 resources and it gives me no way of knowing which one was dragged. In your example you just grab the first one, but it's my understanding that the order doesn't change based on which one was dragged--the dragged one could be the second or third.

**Comment by acerix** [comment]
Yes, I see now there is no way to tell which resource the event is being dragged from when it has more than one.

If your parent resources are fixed height, it may work to use `jsEvent.clientY` from `eventDragStart` to tell the parent resource based on y position.
